packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -267,8 +271,17 @@
         .dc-dropdown__display {
             height: 4rem;
 
-            .dc-dropdown__display-placeholder-text {
-                color: var(--text-less-prominent);
+            .dc-dropdown__display-placeholder {
+                &-text {
+                    color: var(--text-less-prominent);
+                }
+                &-text-limit-fit {
+                    white-space: nowrap;
+                    text-overflow: ellipsis;
+                    overflow: hidden;
+                    display: inline-block;
+                    max-width: calc(40rem - 3.6rem);
```

Comment:
Is there a reason to use this? Shouldn't it be the same as 36.4rem?

Reply:
Arrow in the input has property absolute. So, we may compute: width container - (space before arrow + width arrow)